scripts/build/build.sh

```diff
@@ -24,6 +24,9 @@ header "Checking Pre-Reqs"
 
 $REPOROOT/scripts/test/check-prereqs.sh
 
+# Increases the file descriptors limit for this bash. It prevents an issue we were hitting during restore
+ulimit -n 512
```

Comment:
512 seems like a reasonable value, but I guess it's not clear if that's per process or machine.

If it's per process this will be fine, but per machine and we may cause some unexpected breakage of other processes.

I put together a quick script for getting the current max value if you wish to use it:
`ulimit -a | grep -Eo "\(-n\)\s+\w+" | grep -Eo "[0-9]+"`